Formula/jc.rb

```diff
@@ -0,0 +1,15 @@
+class Jc < Formula
+  desc "This tool serializes the output of popular gnu linux command-line tools and file types to structured JSON output"
+  homepage "https://github.com/kellyjonbrazil/jc"
+  url "https://jc-packages.s3-us-west-1.amazonaws.com/bin/jc-1.10.5-darwin.tar.gz"
```

Comment:
Formulae should build from source in homebrew-core: https://docs.brew.sh/Acceptable-Formulae#we-dont-like-binary-formulae

For an example of a python app formula, run `brew cat black`